classes/emailTemplate/DAO.php

```diff
@@ -379,7 +379,18 @@ public function installAlternateEmailTemplates(int $contextId, ?string $emailKey
                 ->exists();
 
             if (!$exists) {
-                throw new Exception('Tried to install email template as an alternate to `' . $alternateTo . '`, but no default template exists with this key.');
+                trigger_error(
+                    'Tried to install email template as an alternate to `' . $alternateTo . '`, but no default template exists with this key. Installing ' . $alternateTo . ' email template first',
+                    E_USER_WARNING
+                );
+                try {
+                    $siteDao = DAORegistry::getDAO('SiteDAO'); /** @var SiteDAO $siteDao */
+                    $site = $siteDao->getSite(); /** @var Site $site */
+                    $locales = $site->getInstalledLocales();
+                    $this->installEmailTemplates(Repo::emailTemplate()->dao->getMainEmailTemplatesFilename(), $locales, $alternateTo);
+                } catch (Exception $e) {
+                    error_log($e->getMessage());
```

Comment:
If the previous behavior was right, regarding not executing the next instruction if the `$alternateTo` entry doesn't exist, then I think the try/catch should be removed, to allow the exception to leak.